lib/si-events-rs/src/audit_log/v1.rs

```diff
@@ -368,8 +717,26 @@ pub enum AuditLogMetadataV1 {
         after_secret_id: Option<SecretId>,
     },
     #[serde(rename_all = "camelCase")]
+    UpdateSchemaVariant {
+        old_display_name: String,
+        new_display_name: String,
+        old_description: String,
+        new_description: String,
+        old_category: String,
+        new_category: String,
+        old_link: String,
+        new_link: String,
+        old_color: String,
+        new_color: String,
+        old_component_type: String,
+        new_component_type: String,
+        //todo: what to do about the code?
```

Comment:
What is this referring to?

Reply:
Just calling out that for updating the code for a schema variant definition, I wasn't sure how to handle that and punted :)

Reply:
Gotcha!